src/console/plugin.ts

```diff
@@ -84,10 +88,16 @@ const LANDSCAPE_ICON_CLASS = 'jp-MainAreaLandscapeIcon';
  */
 const CONSOLE_ICON_CLASS = 'jp-ImageConsole';
 
+
 /**
- * The file extension for consoles.
+ * The interface for a start console.
```

Comment:
The phrase "start console" is a verb, but we are using it here as a noun. Maybe rename the class to something that is more noun like? Maybe "ConsoleStarter"?